feat(chromium): roll Chromium to r848005

sadym-chromium opened the pull request:
This corresponds to Chromium 90.0.4403.0
This roll includes:
- Cut screenshot by ViewPort size, not position (crrev.com/c/2643792)

BREAKING CHANGE:
- `page.screenshot` cuts screenshot content by the ViewPort size, not ViewPort position.

sadym-chromium committed:
Roll Chrome 90

sadym-chromium committed:
Update screenshot test

sadym-chromium committed:
Add screenshot artefacts tests

sadym-chromium committed:
Add link to crbug

sadym-chromium committed:
update firefox screenshot

sadym-chromium committed:
Merge branch 'main' into chrome_90

mathiasbynens commented:
The images in this patch are much larger than the current versions:

![](https://user-images.githubusercontent.com/81942/106594629-a8e16700-6552-11eb-984f-1ad37f7af343.png)

Please compress them using ImageOptim or Squoosh.

LGTM otherwise

sadym-chromium committed:
eslint

sadym-chromium committed:
make test images smaller

sadym-chromium committed:
compress test images

sadym-chromium commented:
Compressed all 4 images, both new and old.

sadym-chromium committed:
Merge branch 'main' into chrome_90

sadym-chromium committed:
Update versions.js

sadym-chromium committed:
dummy commit

sadym-chromium committed:
feat(chromium): roll Chromium to r848005

This corresponds to Chromium 90.0.4403.0
This roll includes:
- Cut screenshot by ViewPort size, not position (crrev.com/c/2643792)

BREAKING CHANGE:
- `page.screenshot` cuts screenshot content by the ViewPort size, not ViewPort position.
